Mathlib/Algebra/Category/ModuleCat/Presheaf/ChangeOfRings.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Algebra.Category.ModuleCat.ChangeOfRings
+import Mathlib.Algebra.Category.ModuleCat.Presheaf
+
+/-!
+# Change of presheaf of rings
+
+In this file, we define the restriction of scalars functor
+`restrictScalars α : PresheafOfModules.{v} R' ⥤ PresheafOfModules.{v} R`
+attached to a morphism of presheaves of rings `α : R ⟶ R'`.
+
+-/
+
+universe v v' u u'
+
+open CategoryTheory
+
+namespace PresheafOfModules
+
+variable {C : Type u'} [Category.{v'} C] {R R' : Cᵒᵖ ⥤ RingCat.{u}}
+
+/-- The restriction of scalars of presheaves of modules, on objects. -/
+@[simps]
+noncomputable def restrictScalarsBundledCore (M' : PresheafOfModules R') (α : R ⟶ R') :
```

Comment:
It could be, but the file is very short, and doing so would require making also `M'` a global variable, otherwise the order of the parameters would have to be changed.